source/_integrations/unifiprotect.markdown

```diff
@@ -77,6 +79,25 @@ Each UniFi Protect camera will get the following entities added:
 
 * **Camera** - A camera entity for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
 * **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](https://www.home-assistant.io/integrations/tts/#service-say).
+* **Button** - A disabled by default button entity is added for each camera entity. The button will let you reboot your camera device.
+
+### Floodlight
+
+Each UniFi Protect flood light will get the following entities added:
+
+* **Button** - A disabled by default button entity is added for each floodlight entity. The button will let you reboot your floodlight device.
+
+### Sensor
+
+Each UniFi Protect sensor will get the following entities added:
+
+* **Button** - A disabled by default button entity is added for each sensor entity. The button will let you reboot your sensor device.
```

Comment:
Same as above. I find this rather confusing actually... is it a button or a sensor?